app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Good job so far 👏 However, as it is in the project requirements you are supposed to add `:confirmable` attribute together with others 😃 👍 this helps for email confirmation process 👍